common-libs/did-auth-lib/src/shared/Signer.ts

```diff
@@ -0,0 +1,42 @@
+import { KeysService, KeyVault } from '@affinidi/common'
+import base64url from 'base64url'
+
+type SignerOptions = {
+  did: string
+  keyId: string
+  keyVault: KeyVault
+}
+
+export default class Signer {
```

Comment:
This sounds very abstract, though it's a bit tied to the "JWT"? Perhaps we can call it "JwtSigner"?